src/java.base/share/classes/javax/crypto/spec/HKDFParameterSpec.java

```diff
@@ -423,7 +428,7 @@ final class ExtractThenExpand implements HKDFParameterSpec {
          *     {@code null}); the byte[] is copied to prevent subsequent
          *     modification
          * @param length
-         *     the length of the output key material (must be > 0 and < 255 *
+         *     the length of the output keying material (must be > 0 and < 255 *
```

Comment:
Maybe you don't need to talk about the length restriction here. The rule is at input.